sdk/resourcemanager/Azure.ResourceManager/src/BicepModelReaderWriterOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager
+{
+    /// <summary>
+    /// Provides the options for reading and writing bicep.
+    /// </summary>
+    public class BicepModelReaderWriterOptions : ModelReaderWriterOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="BicepModelReaderWriterOptions"/>.
+        /// </summary>
+        public BicepModelReaderWriterOptions() : base("bicep")
```

Comment:
Will we ever want to distinguish between vanilla bicep and CDK bicep?

Nit - consider either adding a `const` or using `format: "bicep"` here so it's clearer what that's being used for in the base .ctor.

Reply:
I don't think so. There isn't any other way to represent these objects in bicep.